modules/pam_motd/pam_motd.c

```diff
@@ -199,8 +194,7 @@ static void try_to_display_directories_with_overrides(pam_handle_t *pamh,
 
     for (i = 0; i < num_motd_dirs; i++) {
 	int rv;
-	rv = scandir(motd_dir_path_split[i], &(dirscans[i]),
-		filter_dirents, alphasort);
+        rv = scandir(motd_dir_path_split[i], &(dirscans[i]), NULL, NULL);
```

Comment:
What was the rationale behind this `alphasort` removal?

Reply:
That `alphasort` was always superfluous, the final sorting happens later anyway, by `qsort`.